evals/benchmark/data_zh.txt

```diff
@@ -0,0 +1,100 @@
+耐克公司的主要业务活动是什么？
```

Comment:
I would suggest to remove this from the PR. the queries are only for internal test purpose. we just need ensure the script has interface to support such usage.